extensions/src/main/java/org/pentaho/platform/web/http/api/resources/UserRoleListResource.java

```diff
@@ -233,11 +234,11 @@ public RoleListWrapper getRoles( @QueryParam( "addExtraRoles" ) String addExtraR
   }
 
   /**
-   * Get a list of the all roles in the platform including Anonymous and Administrator.
+   * Get a list of the all roles in the platform including Administrator.If excludeAnonymous is false, Anonymous role is included, else excluded.
```

Comment:
Missing a space after the first period.
Also, the `@param` doclet is missing.

Reply:
Addressed